docs/LearnCalico.md

```diff
@@ -0,0 +1,351 @@
+these changes, then triggers BIRD to load the new files.
+
+## Lifecycle of a Calico networked container
+
+### Start the calico/node instance on your container hosts
+
+With two hosts on the same network, on each host, the `calicoctl node` command 
+is run.  This creates a container called `calico-node`, which uses the 
+`calico/node` Docker image.
+
+At this point we have a Felix agent, a BIRD BGP daemon, and confd running in the 
+calico-node container on each host. The `calicoctl` command stores the default 
+global AS number and the IP address of each host as BGP host addresses in etcd. 
+The confd process reads these BGP IP addresses and uses a template to configure 
+the BGP daemon with BGP peers.  The two hosts are BGP peers connected in a full 
+mesh using the default global AS number.
+
+There are currently no workload containers running and Felix is standing by, 
+waiting for changes in the etcd datastore.
+
+![calicoctl node](diagrams/calicoctl_node.png)
+
+calico-docker allows you to configure other BGP topologies as well, such as a 
+mesh of route reflectors or per-node BGP.  Visit the calico-docker [BGP 
+page](https://github.com/projectcalico/calico-docker/blob/master/docs/bgp.md) 
+for more details.
+
+### Create Containers on Host
+
+Two containers are created using the orchestrator (such as running
+`docker run --name=workload_A -tid ubuntu`).  The workloads have not been added 
+to Calico networking and no security policy has been configured at this point, 
+so they do not yet have connectivity or Calico-assigned IP addresses.  
+
+The workloads do not have connectivity between one another, but they can access 
+the internet via the docker bridge interface. None of the Calico processes have 
+performed any new actions at this point.
+
+![docker run](diagrams/docker_run.png)
+
+**Note**: Docker's libnetwork plugin works a bit differently than this 
+default-networking example. When using Calico with libnetwork, this step and the 
+next three steps in the process all occur at the same time.  The container and 
+profile are created simultaneously, the container is added to Calico networking, 
+and the profile is set on the container.  Continue reading to learn more about 
+what happens under the covers.
+
+### Add Containers to Calico Networking
+
+The workloads are added to Calico networking by calling:
+
+    calicoctl container add <container_id> <ip_address>
+
+The `<container_id>` can be either the name of the container or the container's 
+workload id.  The `<ip_address>` must be an address that falls within a 
+configured IP pool.  The default IPv4 pool for calico is `192.168.0.0/16`. 
+You can view pools by running `calicoctl pool show`. To add a new pool, run 
+`calicoctl pool add <cidr>` or `calicoctl pool range add <start_ip> <end_ip>`. 
+Etcd stores pools as `/calico/v1/ipam/v4/pool/192.168.0.0-16`, where assigned 
+IPs in a pool are stored as: `/calico/v1/ipam/v4/assignment/192.168.0.0-16/192.168.1.1`.
```

Comment:
The examples in this paragraph all use IPv4. Am I limited to IPv4 pools and addresses, or could I use IPv6 instead?